source/sync/data-access-patterns/flexible-sync-permissions-guide.txt

```diff
@@ -0,0 +1,607 @@
+           "write": {"owner_id": "%%user.id"}
+         }
+       ]  
+     }
+
+  Note that the "read" and "write" expressions use the ``owner_id`` field we
+  marked as "queryable" above. It also uses the ``%%user`` "expansion" to read
+  the requesting user's id. Realm replaces (or "expands") the expansion at
+  evaluation time with the corresponding value -- in this case, the user object.
+
+  When a client accesses the data through Sync, Realm evaluates these
+  expressions to determine permissions. This expression says the requesting user
+  may read or write if the document's ``owner_id`` field is equal to the user's
+  ID.
+
+- Skip :guilabel:`Advanced Configuration`, and then click :guilabel:`Save Changes` 
+  to enable Sync with these permissions.
+
+
+.. _write-own-data-read-all-data:
+
+Write Own Data, Read All Data
+-----------------------------
+
+In this case, users can read all data, but write only their own data. Consider a 
+recipe app where users can read all recipes and add new recipes. The recipes 
+they add can be viewed by everyone using the app. 
+
+To set this up, follow these steps:
+
+.. include:: /includes/use-flex-sync-templates
+
+- :guilabel:`Define Permissions`: use the template dropdown to
+  select the template called "Users can read all data but only write their own data".
+
+  This populates the rule expression box with the following:
+
+  .. code-block:: json
+     :emphasize-lines: 6-7
+  
+     {
+        "defaultRoles": [
+          {
+            "name": "owner-write", 
+            "applyWhen": {},
+            "read": true,
+            "write": {"owner_id": "%%user.id"}
+          }    
+        ]  
+      }
+
+  Note that the "read" expression is set to ``true``, indicating that no matter
+  which user is authenticated, they can access all of the data. The "write"
+  expression uses the ``owner_id`` field we marked as "queryable" above and uses
+  the ``%%user`` "expansion" to match against the requesting user's id. Realm
+  replaces (or "expands") the expansion at evaluation time with the
+  corresponding value -- in this case, the user object.
+
+  When a user accesses the data through Sync, Realm evaluates these
+  expressions to determine permissions. This expression says the requesting user
+  may read all data, but write only if the document's ``owner_id`` field is equal 
+  to the user's ID.
+
+- Skip :guilabel:`Advanced Configuration`, and then click :guilabel:`Save Changes` 
+  to enable Sync with these permissions.
+
+.. _admin-privileges:
+
+Administrator Privileges
+------------------------
+
+In this permission strategy, users with a specific "administrator" role can read
+and write any document. Users who do not have the specified role can only read
+and write their own data. To make this strategy work, you first need to define
+which users have administrator permissions.
+
+MongoDB Realm has a feature called Custom User Data that allows you to associate
```

Comment:
I know we don't want to "wikipedia style" link everywhere in the docs any more, but as a reader I really wanted to know more about Custom User Data here. Maybe we could add a see also at the end of this section linking to the Custom User Data docs?

Reply:
The idea is that this text walks you through setting it up so I wanted to be sure readers knew we were about to explain the steps. This is contextualizing. I think we could definitely add a "Read more: Custom User Data" here though.

Reply:
+1 link custom user data